Add Python and Go write instructions to Cloud Dedicated get started (#4876)
influxdata/docs-v2: content/influxdb/cloud-dedicated/get-started/write.md

## Write line protocol to InfluxDB

Use the **InfluxDB v2 HTTP API** or **InfluxDB client libraries** to write the
line protocol above to InfluxDB.

> Examples are provided for the **Python** and **Go** client libraries.

Reviewer: ```suggestion
The following examples show how to use the **Python** and **Go** client libraries to write line protocol.
```

    [line protocol above](#home-sensor-data-line-protocol) to InfluxDB.

    1.  Import the following packages

        - `context`
        - `fmt`
        - `log`
        - `os`
        - `time`
        - `github.com/influxdata/influxdb-client-go/v2` aliased as `influxdb2`
    
    2.  Create a `dbWrite` function.
    3.  In the `dbWrite` function, use `influxdb2.NewClientWithOptions` to
        create a `writeClient` that accepts write options.
        The write client requires the following credentials:
        
        - **url**: InfluxDB Cloud Dedicated cluster URL
        - **token**: [Database token](/influxdb/cloud-dedicated/admin/tokens/)
          with write access to the **get-started** database.
          _For security reasons, we recommend setting this as an environment
          variable rather than including the raw token string._

        Because the timestamps in the line protocol are in second
        precision, **use `SetPrecision(time.Second)` to define the write precision option**.
    
    4.  Use `writeClient.WriteAPIBlocking` to define a `writeAPI`.
        The write API requires the following credentials:

>         - **org**: _Leave as an empty string_

Author: ```suggestion
        - **org**: _Provide an arbitrary string. This credential is ignored._
```

        - **bucket**: InfluxDB database name.
    
    5.  Define an array of line protocol strings where each element is a single
        line of line protocol.
    
    6.  Iterate through the array of line protocol and use `writeAPI.WriteRecord`
        to write each line of line protocol to InfluxDB.
    
    7.  Define a `main` function that executes the `dbWrite` function.

{{% influxdb/custom-timestamps %}}

```go
package main

import (
	"context"
	"fmt"
	"log"
	"os"
	"time"

	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
)

func dbWrite(ctx context.Context) error {
	// Create write client

> 	url := "https://us-east-1-1.aws.cloud2.influxdata.com"

Author: ```suggestion
	url := "https://cloud2.influxdata.com"
```

	token := os.Getenv("INFLUX_TOKEN")
	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))

	// Define write API

> 	org := ""

Author: ```suggestion
	org := "ignored"
```

          with write access to the **get-started** database.
          _For security reasons, we recommend setting this as an environment
          variable rather than including the raw token string._

        Because the timestamps in the line protocol are in second
        precision, **use `SetPrecision(time.Second)` to define the write precision option**.
    
    4.  Use `writeClient.WriteAPIBlocking` to define a `writeAPI`.
        The write API requires the following credentials:

        - **org**: _Leave as an empty string_
        - **bucket**: InfluxDB database name.
    
    5.  Define an array of line protocol strings where each element is a single
        line of line protocol.
    
    6.  Iterate through the array of line protocol and use `writeAPI.WriteRecord`
        to write each line of line protocol to InfluxDB.
    
    7.  Define a `main` function that executes the `dbWrite` function.

{{% influxdb/custom-timestamps %}}

```go
package main

import (
	"context"
	"fmt"
	"log"
	"os"
	"time"

	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
)

func dbWrite(ctx context.Context) error {
	// Create write client

> 	url := "https://us-east-1-1.aws.cloud2.influxdata.com"

Reviewer: Should this be `https://cloud2.influxdata.com`?

	token := os.Getenv("INFLUX_TOKEN")
	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))

	// Define write API

> 	org := ""

Reviewer: In my experience, `org` can't be empty.

Author: K, I'll note that this should be an arbitrary string.

> 	bucket := "<YOUR_INFLUX_DATABASE>"

Reviewer: ```suggestion
	bucket := "DATABASE_NAME"
```
